.gitignore

```diff
@@ -4,3 +4,6 @@ docs
 .nyc_output
 coverage
 npm-debug.log
+package-lock.json
+yarn.lock
```

Comment:
I don't know if we want to add `yarn.lock` into `.gitignore` right now given that it's going to be committed in v6.